Packs/DSPM/Integrations/DSPM/DSPM.py

```diff
@@ -0,0 +1,903 @@
+
+    alerts_list = client.get_alerts_list(params)
+    return alerts_list
+
+
+def get_integration_config():
+
+    integration_config = {
+        "dspmApiKey": demisto.params().get("dspmApiKey", {}).get("password"),
+        "slackMsgLifetime": demisto.params().get("slackMsgLifetime"),
+        "defaultSlackUser": demisto.params().get("defaultSlackUser"),
+    }
+    demisto.debug(f" integration config : ${integration_config}")
+
+    # Prepare data for table format
+    table_data = [
+        {"Key": key, "Value": value} for key, value in integration_config.items()
+    ]
+
+    # Convert dictionary to table format
+    markdown = tableToMarkdown(
+        "Integration Configuration", table_data, headers=["Key", "Value"]
+    )
+
+    # Return CommandResults with the integration config
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix="DSPM.IntegrationConfig",
+        outputs_key_field="config",
+        outputs={"integration_config": integration_config},
+    )
+
+
+""" MAIN FUNCTION """
+
+
+def main() -> None:
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+
+    # get the service API url
+    base_url = demisto.params().get("dspmBaseUrl")
+    api_key = demisto.params().get("dspmApiKey", {}).get("password")
+    verify_certificate = not demisto.params().get("insecure", False)
+    proxy = demisto.params().get("proxy", False)
+
+    demisto.debug(f"Command being called is {demisto.command()}")
+    try:
+        client = Client(
+            base_url=base_url, api_key=api_key, verify=verify_certificate, proxy=proxy
+        )
+
+        if demisto.command() == "test-module":
+            result = test_module(client)
+            return_results(result)
+        elif demisto.command() == "dspm-get-integration-config":
+            return_results(get_integration_config())
+        elif demisto.command() == "dspm-get-risk-findings":
+            page = 0
+            while True:
+                findings = dspm_get_risk_findings(client, demisto.args(), page)
+                if not findings:
+                    if page == 0:
+                        demisto.info("No risks were fetched")
+                        readable_output = "### Risk Findings\n **No entries.**\n"
+                        return_results(
+                            CommandResults(
+                                outputs_prefix="DSPM.RiskFindings",
+                                outputs_key_field="id",
+                                readable_output=readable_output,
+                            )
+                        )
+                    break  # No more findings to fetch
+
+                return_results(
+                    CommandResults(
+                        outputs_prefix="DSPM.RiskFindings",
+                        outputs_key_field="id",
+                        outputs=findings,
+                    )
+                )
+                page += 1
```

Comment:
please move this logic to different method
do the same for :
- dspm-get-list-of-assets
- dspm-get-data-types-findings
- dspm-get-list-of-alerts

Reply:
Yes, we have moved the logic to a different method as you suggested.